client/src/controllers/TeleportController.test.js

```diff
@@ -99,21 +99,80 @@ describe('TeleportController', () => {
       const shadowRoot = shadowHost.attachShadow({ mode: 'open' });
       document.body.appendChild(shadowHost);
 
+      shadowRoot.innerHTML = /* html */ `
+        <div>
+          <aside>
+            <!--
+            The template will be moved here, and then the content will be
+            teleported to the <div> (first child of the shadow root)
+            when the app starts
+            -->
+          </aside>
+        </div>
+      `;
+
+      // Move the template to the aside element
+      const aside = shadowRoot.querySelector('aside');
       const template = document.getElementById('template');
-      const content = template.content.cloneNode(true);
+      aside.append(template);
 
-      const targetContainer = document.createElement('div');
-      targetContainer.setAttribute('id', 'target-container');
-      targetContainer.appendChild(content);
-      shadowRoot.appendChild(targetContainer);
```

Comment:
This original test is bogus, as it actually clones the template content and append it to the target container i.e. doing what the controller is supposed to do. By the end of the test, the template element still exists (instead of removed).

Reply:
Thanks for finding this issue.